test/mockupdb/test_query_read_pref_sharded.py

```diff
@@ -30,7 +30,7 @@ class TestQueryAndReadModeSharded(unittest.TestCase):
     def test_query_and_read_mode_sharded_op_query(self):
```

Comment:
This test is for op-query so we can remove it. There's already an op_msg test below.